models/multi_organ_segmentation/configs/train.yaml

```diff
@@ -0,0 +1,354 @@
+    num_workers: 4
+  inferer:
+    _target_: SimpleInferer
+  postprocessing:
+    _target_: Compose
+    transforms:
+    - _target_: Activationsd
+      keys: pred
+      softmax: true
+    - _target_: AsDiscreted
+      keys:
+      - pred
+      - label
+      argmax:
+      - true
+      - false
+      to_onehot: "@output_classes"
+  handlers:
+  - _target_: LrScheduleHandler
+    lr_scheduler: "@lr_scheduler"
+    print_lr: true
+  - _target_: ValidationHandler
+    validator: "@validate#evaluator"
+    epoch_level: true
+    interval: "@val_interval"
+  - _target_: StatsHandler
+    tag_name: train_loss
+    output_transform: "$monai.handlers.from_engine(['loss'], first=True)"
+  - _target_: TensorBoardStatsHandler
+    log_dir: "@output_dir"
+    tag_name: train_loss
+    output_transform: "$monai.handlers.from_engine(['loss'], first=True)"
+  key_metric:
+    train_accuracy:
+      _target_: ignite.metrics.Accuracy
+      output_transform: "$monai.handlers.from_engine(['pred', 'label'])"
+  trainer:
+    _target_: SupervisedTrainer
+    max_epochs: 400
+    device: "@device"
+    train_data_loader: "@train#dataloader"
+    network: "@network"
+    loss_function: "@loss"
+    optimizer: "@optimizer"
+    inferer: "@train#inferer"
+    postprocessing: "@train#postprocessing"
+    key_train_metric: "@train#key_metric"
+    train_handlers: "@train#handlers"
+    amp: true
+validate:
+  preprocessing:
+    _target_: Compose
+    transforms: "%train#deterministic_transforms"
+  dataset:
+    _target_: CacheDataset
+    data: "@val_datalist"
+    transform: "@validate#preprocessing"
+    cache_rate: 0.125
+  dataloader:
+    _target_: DataLoader
+    dataset: "@validate#dataset"
+    batch_size: 1
+    shuffle: false
+    num_workers: 4
+  inferer:
+    _target_: SlidingWindowInferer
+    roi_size:
+    - 96
+    - 96
+    - 96
+    sw_batch_size: 6
+    overlap: 0.625
+  postprocessing: "%train#postprocessing"
+  handlers:
+  - _target_: StatsHandler
+    iteration_log: false
+  - _target_: TensorBoardStatsHandler
+    log_dir: "@output_dir"
+    iteration_log: false
+  - _target_: CheckpointSaver
+    save_dir: "@ckpt_dir"
+    save_dict:
+      model: "@network"
+    save_key_metric: true
+    key_metric_filename: model.pt
+  key_metric:
+    val_mean_dice:
+      _target_: MeanDice
+      include_background: false
+      output_transform: "$monai.handlers.from_engine(['pred', 'label'])"
+  additional_metrics:
+    val_accuracy:
+      _target_: ignite.metrics.Accuracy
+      output_transform: "$monai.handlers.from_engine(['pred', 'label'])"
+  evaluator:
+    _target_: SupervisedEvaluator
+    device: "@device"
+    val_data_loader: "@validate#dataloader"
+    network: "@network"
+    inferer: "@validate#inferer"
+    postprocessing: "@validate#postprocessing"
+    key_val_metric: "@validate#key_metric"
+    additional_metrics: "@validate#additional_metrics"
+    val_handlers: "@validate#handlers"
+    amp: true
+training:
```

Comment:
Needs to follow the "initialize", "run", "finalize" schema.

Reply:
Update the configs based on the latest schema.